tests/snapshots/repo__repo_with_token_url.snap

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Tried to disable colors, but I forgot that using empty custom colors is equivalent to passing nothing and letting the default colors get used. But a colored snapshot test isn't terrible, I guess. Looks pretty with `cat path/to/snapshot` :laughing: